src/view-mixin.js

```diff
@@ -208,26 +188,29 @@ var AbstractView = Backbone.View.extend({
 
     this.triggerMethod.apply(this, ['before:destroy'].concat(args));
 
-    // mark as destroyed before doing the actual destroy, to
-    // prevent infinite loops within "destroy" event handlers
-    // that are trying to destroy other views
+    // update lifecycle flags
     this._isDestroyed = true;
-    this.triggerMethod.apply(this, ['destroy'].concat(args));
+    this._isRendered = false;
 
     // unbind UI elements
     this.unbindUIElements();
 
-    this._isRendered = false;
-
     // remove the view from the DOM
-    this.remove();
+    https://github.com/jashkenas/backbone/blob/1.2.3/backbone.js#L1235
+    this._removeElement();
 
+    // remove children after the remove to prevent extra paints
+    this._removeChildren();
```

Comment:
removeChildren was made as a generic call that collection view could use to destroy its children and view could use to remove its regions.  This allowed for `destroy` to be shared where it is otherwise identical.  Alternatively I could see just setting up a listener for destroy inside of `_initRegions` and `_initBehaviors` which might simplify things a bit more.  Thoughts?